ui/lib/apps/InstanceProfiling/pages/Detail.tsx

```diff
@@ -10,11 +10,29 @@ import { CardTable, DateTime, Head, Descriptions } from '@lib/components'
 import { useClientRequestWithPolling } from '@lib/utils/useClientRequest'
 import { InstanceKindName } from '@lib/utils/instanceTable'
 import useQueryParams from '@lib/utils/useQueryParams'
+import { ScrollablePane } from 'office-ui-fabric-react'
+import { MenuInfo } from 'rc-menu/lib/interface'
 
-function mapData(data) {
+enum ViewOutputTypeOptions {
+  FlameGraph = 'instance_profiling.detail.table.columns.selection.flamegraph',
+  Graph = 'instance_profiling.detail.table.columns.selection.graph',
+}
+
+enum taskState {
+  Error,
+  Running,
+  Success,
+}
+
+const ProfilingOutputTypeProtobuf = 'protobuf'
+
+let viewDefaultOutputTypeVal = ViewOutputTypeOptions.FlameGraph
```

Comment:
I think there is no "default to flame graph" or something. What we only have is `FlameGraph is preferred over Graph for protobuf data`. This definition can be very misleading.